src/main/java/net/minecraftforge/event/entity/living/LivingSpawnEvent.java

```diff
@@ -116,9 +141,27 @@ public CheckSpawn(EntityLiving entity, World world, float x, float y, float z)
     @Cancelable
     public static class SpecialSpawn extends LivingSpawnEvent
     {
+        private final @Nullable MobSpawnerBaseLogic spawner;
```

Comment:
`@Nullable` should go on the line before, except in patches where size is being minimized.